ambry-messageformat/src/test/java/com.github.ambry.messageformat/MessageFormatInputStreamTest.java

```diff
@@ -215,12 +233,20 @@ private void messageFormatRecordsTest(short blobVersion, BlobType blobType, bool
             blobType);
 
     int totalSize;
-    if (useV2Header) {
-      totalSize = headerSize + key.sizeInBytes() + blobEncryptionKeySize + blobPropertiesRecordSize + userMetadataSize
-          + (int) blobSize;
-    } else {
-      totalSize = headerSize + key.sizeInBytes() + blobPropertiesRecordSize + userMetadataSize + (int) blobSize;
+    switch (headerVersion) {
+      case MessageFormatRecord.Message_Header_Version_V1:
+        totalSize = headerSize + key.sizeInBytes() + blobPropertiesRecordSize + userMetadataSize + (int) blobSize;
+        break;
+      default: //case MessageFormatRecord.Message_Header_Version_V2:
```

Comment:
add todo here? I believe there will be V3 case in future test

Reply:
done